app/classifier/frame-annotator.cjsx

```diff
@@ -23,8 +23,10 @@ module.exports = React.createClass
     onChange: Function.prototype
 
   getInitialState: ->
-    naturalWidth: 0
-    naturalHeight: 0
+    naturalWidth: @props.naturalWidth
+    naturalHeight: @props.naturalHeight
```

Comment:
props should be immutable, so storing `naturalWidth` and `naturalHeight` as props and state looks wrong to me. If they don't change, then you can remove the state variables.